add_item_form.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+$db = new PDO ('mysql:host=db;dbname=collection', 'root', 'password');
+
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$nameQuery = $db->prepare( 'SELECT `name` FROM `games`;');
```

Comment:
Thought you were leaving yourself a note for later but this is the name 😆 Love it!